charts/github-bot-safe-settings/templates/app-env.yaml

```
@@ -0,0 +1,22 @@
+{{- $fullName := include "safe-settings.fullname" . -}}
+{{- if .Values.appEnv.create }}
```

Let's call it `appEnvSecret`, i.e.

```suggestion
{{- if .Values.appEnvSecret.create }}
```

Also let's rename this file to `app-env-secret.yaml`

---

resolve cause it is fork